lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -185,14 +185,14 @@ bool LLDBMemoryReader::readBytes(swift::remote::RemoteAddress address,
   llvm::Optional<Address> maybeAddr =
       resolveRemoteAddress(address.getAddressData());
   if (!maybeAddr) {
-    LLDB_LOGV(log, "[MemoryReader] could not resolve address {1:x}",
+    LLDB_LOG(log, "[MemoryReader] could not resolve address {1:x}",
```

Comment:
@kastiglione Since these are indicative of errors, should this maybe even be promoted to a HEALTH log?

Reply:
given that we don't expect this failure to happen, surfacing it for more visibility is a good call